static/static.js

```diff
@@ -2,61 +2,15 @@ var packages = require("./packages");
 require("./styles/styles.less!");
 window.PACKAGES = packages;
 
-(function(){
```

Comment:
Are we sure we don't need all this code you deleted?

Reply:
This was accidental, i've re-added the code and formatted it.